packages/openapi-v3/src/__tests__/integration/controller-spec.integration.ts

```diff
@@ -203,6 +208,249 @@ describe('controller spec', () => {
     });
   });
 
+  context('reference models via spec', () => {
+    it('allows operations to provide definitions of referenced models through #/components/schema', () => {
+      class MyController {
+        @get('/todos', {
+          responses: {
+            '200': {
+              description: 'Array of Category model instances',
+              content: {
+                'application/json': {
+                  schema: {
+                    $ref: '#/components/schemas/Todo',
+                    definitions: {
```

Comment:
Can we add a test case to show that `definitions` in `find` method can be referenced by another method, such as `findById`?

Reply:
I'm not sure how to test this, but since it's verified that the definition is now present in `#/components/schemas`, then is it safe to assume that any method that references `#/components/schemas/Todo` will be able to reference the definition? If not, any suggestions on how to modify [this test](https://github.com/strongloop/loopback-next/pull/2898/commits/3eec6a51b19cc8b775425b2206dc20df3a341049#diff-28766b6f539bb8743b1be35f2ef35b68R310) to test this scenario (I'll remove the test, otherwise)?

Reply:
You can have two methods:
1. The 1st one has `definitions`
2. The 2nd one has `$ref` to a schema in `definitions` from the 1st method